crates/rolldown_binding/src/types/binding_watcher_event.rs

```diff
@@ -93,8 +94,8 @@ pub struct BindingBundleEndEventData {
 #[napi]
 impl BindingBundleEndEventData {
   #[napi(getter)]
-  pub fn result(&self) -> crate::binding_bundler_impl::BindingBundlerImpl {
```

Comment:
Holy crap. You could get `RollupBuild` instance from watch events. I wouldn't notice this if I don't try to remove `BindingBundlerImpl`.

@sapphi-red Do you think it's ok we use `BindingWatcherBundler` for workaround and refactor it when solving https://github.com/rolldown/rolldown/issues/6482 ?

Read https://github.com/rolldown/rolldown/pull/6888#discussion_r2505383004 for continuing

Reply:
Yes, I think it's ok for now with the workaround and leave it for later.